src/selectors-profile.js

```diff
@@ -0,0 +1,110 @@
+import csstree from 'css-tree'
+
+var pseudoSelectorsToKeep = [
+  ':before',
+  ':after',
+  ':visited',
+  ':first-letter',
+  ':first-line'
+]
+// detect these selectors regardless of whether one or two semi-colons are used
+var pseudoSelectorsToKeepRegex = pseudoSelectorsToKeep
+  .map(function (s) {
+    return ':?' + s
+  })
+  .join('|') // separate in regular expression
+// we will replace all instances of these pseudo selectors; hence global flag
+var PSUEDO_SELECTOR_REGEXP = new RegExp(pseudoSelectorsToKeepRegex, 'g')
+
+// TODO: bind with forceInclude argument instead
```

Comment:
comment irrelevant now (had to do with execution in the sandboxed browser environment previously)

Reply:
The same as above, I deleted this comment